SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -428,7 +428,7 @@ bool USpatialLatencyTracer::ContinueLatencyTrace_Internal(const AActor* Actor, c
 	}
 
 	// We do minimal internal tracking for native rpcs/properties
-	const bool bInternalTracking = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking() || Type == ETraceType::Tagged;
+	const bool bInternalTracking = Type == ETraceType::Tagged;
```

Comment:
Why has this changed?

Reply:
This is the `fix GDK latency tests` additional that I requested as mentioned in description - one line change that didn't need to be in this PR but means we don't have to wait another 45 mins to cut the rc.